core/src/main/java/com/linecorp/armeria/common/stream/AbstractStreamMessageWriter.java

```diff
@@ -26,7 +26,7 @@
 import com.linecorp.armeria.internal.common.stream.AbortingSubscriber;
 import com.linecorp.armeria.internal.common.stream.StreamMessageUtil;
 
-abstract class AbstractStreamMessageWriter<T> extends CancellableStreamMessage<T> implements StreamWriter<T> {
+abstract class AbstractStreamWriter<T> extends CancellableStreamMessage<T> implements StreamWriter<T> {
```

Comment:
Oops, it seems like the file name isn't changed. 😆

Reply:
https://github.com/line/armeria/blob/e7b02d4c1d758f90180fd22686f105ad82fb731a/core/src/main/java/com/linecorp/armeria/common/stream/AbstractStreamMessageWriter.java#L29

oh on latest `master` branch, still `AbstractStreamMessageWriter `. 

It's better to rename `AbstractStreamMessageWriter` -> `AbstractStreamWriter`? 
(cause we rename `StreamMessageWrtier` -> `StreamWriter`)

Reply:
or.. I don't rename `AbstractStreamMessageWriter` in previous PR cause it implements Cancellable`StreamMessage` + `StreamWriter` 😅